moved/src/move_execution/tests.rs

```diff
@@ -560,6 +590,74 @@ fn test_deposit_tx() {
     assert_eq!(balance, mint_amount);
 }
 
+#[test]
+fn test_withdrawal_tx() {
+    let genesis_config = GenesisConfig::default();
+    let mut signer = Signer::new(&PRIVATE_KEY);
+    let (_, mut state) = deploy_contract("natives", &mut signer, &genesis_config);
+
+    // 1. Deposit ETH to user
+    let mint_amount = U256::from(123u64);
```

Comment:
We can make filling up the user's account a function, just to have some balance It's used in a few places now.

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/153/commits/eb4db738d998b6b6de0dddbc7359b5d9fc0aa703